helm/aladdin-demo/templates/aladdin-demo-commands.deploy.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ .Chart.Name }}-commands
+  labels:
+    project: {{ .Chart.Name }}
+    name: {{ .Chart.Name }}-commands
+    app: {{ .Chart.Name }}-commands
+    githash: {{ .Values.deploy.imageTag }}
+spec:
+  replicas: 1
+  strategy:
```

Comment:
Do we not need the maxsurge/maxunavailable here? 
I guess maybe it doesn't matter if it's 1 replica

Reply:
The default is 25%, which i guess is okay for this??